docs/reference/kumo.spf/check_host.md

```diff
@@ -0,0 +1,40 @@
+# `kumo.spf.check_host()`
+
+{{since('2024.10.22-??')}}
```

Comment:
please use `dev` as the placeholder for the unreleased version; at release time I make a sweep for these and replace with the version of the newly minted tag.

```suggestion
{{since('dev')}}
```